.travis.yml

```diff
@@ -1,17 +1,22 @@
 language: python
 cache: pip
+node_js:
+  - "7"
```

Comment:
I think you need to remove these two lines before we merge or we turn this Travis file into a node_js tester instead of a Python tester and we would lose the Python2 vs. Python3 tests.

Also, you might want to include the linting changes from https://github.com/Fiware/dataModels/pull/176/files and then we could just drop #176 ...  Your call.